.github/instructions/docs.instructions.md

```
@@ -0,0 +1,14 @@
+---
+
+# rustpython-parser
+
+When working with the `rustpython-parser` crate, follow these steps to view the documentation:
+
+```bash
+cargo doc --no-deps --package rustpython-parser
+```
+
+or search their GitHub repository for source implementation details:
+https://github.com/RustPython/Parser/tree/main/parser
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)